1st-gen/tools/styles/README.md

```diff
@@ -28,27 +28,98 @@ yarn add @spectrum-web-components/styles
+    - `--spectrum-focus-indicator-color`
+    - `--spectrum-neutral-content-color-default`
+
+- **Semantic component tokens** - Named by purpose, referencing scale-specific values:
+    - `--spectrum-breadcrumbs-height: var(--spectrum-component-height-300)` (where `component-height-300` is 48px in medium, 60px in large)
+    - `--spectrum-meter-default-width: var(--spectrum-meter-width)` (where `meter-width` is 192px in medium, 240px in large)
+
+#### Color options
+
+Each system makes a `dark` and `light` color option available. Currently, the Spectrum system also offers `darkest` and `lightest` color options, but these will be deprecated in the future.
+
+Color option files (`light-vars.css` and `dark-vars.css`) contain raw color definitions that differ between themes, as well as semantic color mappings that reference different colors based on the selected theme:
+
+- **Raw color definitions** - Color scale values that differ between light and dark themes:
+    - `--spectrum-gray-50` is white (`255, 255, 255`) in light theme, but dark gray (`29, 29, 29`) in dark theme
+    - `--spectrum-gray-900` is black (`0, 0, 0`) in light theme, but white (`255, 255, 255`) in dark theme
+    - Full color scales for gray, blue, red, orange, yellow, green, and other semantic color families
+
+- **Semantic color mappings** - Named by purpose, referencing different raw colors based on theme:
+    - `--spectrum-background-base-color` references `--spectrum-gray-200` in light, but `--spectrum-gray-50` in dark
+    - `--spectrum-neutral-subdued-background-color-default` references `--spectrum-gray-600` in light, but `--spectrum-gray-400` in dark
+    - Component-specific colors like `--spectrum-neutral-visual-color`, background colors, and visual indicators
+
+#### Scales
+
+Each system has two scale sizes available: `medium` (default) and `large`. The `medium` scale is designed for desktop environments, while `large` is optimized for mobile and touch devices with larger touch targets. Scale files (`medium-vars.css` and `large-vars.css`) contain raw size and spacing definitions that differ between scales, as well as component-specific sizing values:
+
+- **Raw size and spacing definitions** - Values that differ between medium and large scales:
+    - `--spectrum-font-size-200` is `16px` in medium, but `19px` in large
+    - `--spectrum-component-height-100` is `32px` in medium, but `40px` in large
+    - Icon sizes, spacing values, and other dimensional tokens that scale up for larger/touch interfaces
+
+- **Component-specific sizing** - Tokens for component dimensions that vary by scale:
+    - `--spectrum-meter-width` is `192px` in medium, but `240px` in large
+    - `--spectrum-breadcrumbs-height-multiline` is `72px` in medium, but `84px` in large
+    - Component-specific measurements like button heights, field widths, and icon positioning
+
+#### Available systems
+
+Here is a summary of each system and the options available on each:
+
+<sp-table>
```

Comment:
I added a table. It's a nice way to present information but the information being presented seems kind of obvious, so I took it out. I committed it so I could bring it back if a reviewer feels differently about it though. Here's what it looked like:

<img width="979" height="158" alt="Screenshot 2025-11-20 at 3 26 40 PM" src="https://github.com/user-attachments/assets/838c518f-c1c2-4219-a3a5-138df269dad4" />